package.json

```diff
@@ -86,7 +86,7 @@
     "karma-mocha": "0.2.2",
     "karma-safari-launcher": "0.1.1",
     "karma-sauce-launcher": "1.1.0",
-    "karma-sinon-chai": "1.2.0",
+    "karma-sinon-chai": "1.2.3",
```

Comment:
@subhankar-panda please revert changes to `package.json`, `package-lock.json` and `yarn.lock`, otherwise looks good and is ready to merge after.